src/app/lists/combo/simple-combo-cascading/simple-combo-cascading.component.ts

```diff
@@ -0,0 +1,37 @@
+import * as core from '@angular/core';
```

Comment:
Is there a reason why we're importing everything like this?

Reply:
have no idea where all these came from
resolved now